exemplarchat/src/main/resources/chat-development.yaml

```diff
@@ -1,3 +1,3 @@
 kubernetes:
-  my_pod_namespace: none
-  my_pod_name: none
+  my_pod_namespace: default
+  my_pod_name: Localhost
```

Comment:
should this be lowercase? I’ve never seen the string `Localhost` with a `L` and it looks so weird to my eyes